### PR TITLE
hover: 0.43.0 -> 0.46.1

### PR DESCRIPTION
**Pull request by thiagokokada**
###### Motivation for this change

Multiple new releases: https://github.com/go-flutter-desktop/hover/releases/tag/v0.46.1

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/hover/default.nix [hunk 1]
```diff
@@ -87,12 +87,13 @@ in
 buildFHSUserEnv rec {
   name = pname;
   targetPkgs = pkgs: [
+    binutils
```
Comment (thiagokokada) [inline pkgs/development/tools/hover/default.nix:90]: Needed or I would get:

```
$ hover run
hover: Missing config: file hover.yaml not found: open go/hover.yaml: no such file or directory
hover: Downloading engine for platform linux-debug_unopt at version 5babba6c4d25fa237bbf755ab85c9a0c50b3c6ec...
Download completed in 1.58s
Download completed in 1.22s
hover: Failed to strip /home/thiagoko/.cache/hover/engine/linux-debug_unopt/libflutter_engine.so: exec: "strip": executable file not found in $PATH
```

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 108016` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>hover</li>
  </ul>
</details>

File: pkgs/development/tools/hover/default.nix [hunk 2]
```diff
@@ -36,23 +36,23 @@ let
   hover = buildGoModule rec {
     inherit pname version;
 
-    meta = with stdenv.lib; {
+    meta = with lib; {
       description = "A build tool to run Flutter applications on desktop";
       homepage = "https://github.com/go-flutter-desktop/hover";
       license = licenses.bsd3;
-      platforms = platforms.linux ++ platforms.darwin;
```
Comment (thiagokokada) [inline pkgs/development/tools/hover/default.nix:43]: As it current is, it doesn't build on macOS (i.e.: depends on X11).

File: pkgs/development/tools/hover/default.nix [hunk 3]
```diff
@@ -36,23 +36,23 @@ let
...
+      platforms = platforms.linux;
```
Comment (thiagokokada) [inline pkgs/development/tools/hover/default.nix:43]: As it current is, it doesn't work in macOS (i.e.: depends on X11).

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108016` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>hover</li>
</ul>
</details>

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**